ui/components/app/gas-timing/index.scss

```
@@ -1,26 +1,26 @@
 .typography.gas-timing {
-  color: var(--ui-4);
+  color: var(--color-text-alternative);
 
   &--positive {
-    color: var(--success-3);
+    color: var(--color-success-default);
   }
 
   &--positive-V2 {
-    color: var(--success-3);
+    color: var(--color-success-default);
     font-weight: bold;
   }
 
   &--warning {
-    color: var(--alert-3);
+    color: var(--color-warning-default);
```

Ahh this kills me... 
```suggestion
    color: var(--color-secondary-default);
```

---

Yeah for all of those warning colors I was divided between using warning and secondary, I don't know how we could define which to use where